packages/screens/RiotGame/component/GameContentView.tsx

```diff
@@ -24,6 +25,7 @@ export const GameContentView: React.FC<GameContentViewProps> = ({
   containerStyle,
   contentStyle,
   bgImage,
+  bgImageURI,
```

Comment:
why create a second prop and not pass `{ uri: bgImageURI }` at call sites?

Reply:
You are totally right. Removed unneeded bgImageURI